app/translations/eo/LC_MESSAGES/messages.po

```diff
@@ -2,1124 +2,1300 @@ msgid ""
+msgstr "Yer session haes timed oot as thair wus naethin daein"
 
-#: templates/static/accessibility.html:81
-msgid "Non-compliance with the accessibility regulations"
-msgstr ""
+#: templates/errors/session-expired.html:7
+msgid "To help protect your information we have timed you out"
+msgstr "Tae help bield yer wittins we hae timed ye oot"
 
-#: templates/static/accessibility.html:83
-msgid "Non-descriptive form elements. On the ‘Check your answers’ summary page there is an option to “change” items, but the link does not describe where it goes to. Fails on WCAG 2.1 A number 1.3.1."
-msgstr ""
+#: templates/errors/session-expired.html:8
+msgid "You will need to <a href='{url}'>enter your 16-character access code</a> again to continue your census."
+msgstr "Ye maun<a href='{url}'>pit in yer 16-chairacter ingang code</a>agane tae conteenye wi yer heid-coont."
 
-#: templates/static/accessibility.html:85
-msgid "Error messages are too generic. If the text in a field is incorrect or in the wrong format, the answer validation page says the answer is incorrect without explaining what you need to do. Fails on WCAG 2.1 A number 3.3.1."
-msgstr ""
+#: templates/errors/submission-complete.html:3
+msgid "Submission Complete"
+msgstr "Inpit hale frae en tae yin"
 
-#: templates/static/accessibility.html:87
-msgid "Form label issue. On the ‘What is your name?’ page, middle name is optional but this is not clear to the user. Fails on WCAG 2.1 A number 3.3.2."
-msgstr ""
+#: templates/errors/submission-complete.html:6
+msgid "This page is no longer available"
+msgstr "This page bes nae langer appen tae"
 
-#: templates/static/accessibility.html:89
-msgid "Form label issue. When selecting the appropriate relationship, it is possible to ‘Save and continue’ without selecting an option. Fails on WCAG 2.1 A number 2.4.6."
-msgstr ""
+#: templates/errors/submission-complete.html:7
+msgid "Your census has been submitted"
+msgstr "Yer heid-coont haes bin pit in"
 
-#: templates/static/accessibility.html:91
-msgid "On the ‘Is this address correct?’ page, the title may not accurately reflect the content on the page for audio users. The ‘Submit survey’ page does not have a page title. Fails on WCAG 2.1 A number 2.4.2."
-msgstr ""
+#: templates/errors/submission-failed.html:9
+msgid "You can try to <a href='{url}'>submit your census again</a>"
+msgstr "Ye can try tae<a href='{url}'>pit in yer feedbak agane</a>"
 
-#: templates/static/accessibility.html:93
-msgid "Timeout issue. Users are not informed of a time limit and service times out without warning. During the validation process there is no return link on the session timeout page. Fails on WCAG 2.1 A number 2.2.1 and AAA number 2.2.6."
-msgstr ""
+#: templates/errors/submission-failed.html:10
+msgid "If this problem keeps happening, please <a href='{url}'>contact us</a> for help."
+msgstr "Gin this gies ye mair bother, plaise <a href='{url}'> get tha houl o us</a>fur a pull oot."
```

Comment:
Gin this gies ye mair bother, plaise <a href='{url}'> get tha houl o us</a> fur a pull oot.